firmware/hw_layer/drivers/can/can_hw.cpp

```diff
@@ -32,10 +32,9 @@ class CanRead final : protected ThreadController<UTILITY_THREAD_STACK_SIZE> {
 	{
 	}
 
-	void start(CANDriver* device) {
-		m_device = device;
-
+	void tryStart(CANDriver *device) {
 		if (device) {
+			m_device = device;
```

Comment:
why this change?

Reply:
Is unnecessary, leftover from when I was trying to properly override `start` (w/ separate `setDevice` method); can revert.